src/applications/vaos/components/PastAppointmentsDateDropdown.jsx

```diff
@@ -37,6 +38,7 @@ export default function PastAppointmentsDateDropdown({
           if (currentRange !== dateRangeIndex) {
             onChange(dateRangeIndex);
           }
+          focusElement('#queryResultLabel');
```

Comment:
You shouldn't need this, since we're focusing in the PastAppointmentsList component when the new appointments finish loading. Was something not working without this change?

Reply:
Not really. That's the initial state. I thought it should read the label if the user hit the update button repeatedly.

Reply:
Oh, I see. I would put this in an `else` block, then. So that it doesn't read the label text twice when we do actually have to fetch new appointment data.

Reply:
Done.